city_scrapers/spiders/lemoore_city_council.py

```diff
@@ -0,0 +1,128 @@
+from turtle import title
+from city_scrapers_core.constants import CITY_COUNCIL
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parser
+from datetime import datetime, date
+import re
+
+
+class LemooreCityCouncilSpider(CityScrapersSpider):
+    name = "lemoore_city_council"
+    agency = "Lemoore City Council"
+    timezone = "America/Chicago"
+    start_urls = ["https://lemoore.com/councilagendas"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css("div[id='elementor-tab-content-7431'] table tr")[1:]:
+            date = item.css("td:nth-child(1)::text").get()
+            if date.strip() and "CX" not in date:
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description=self._parse_description(item),
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(item),
+                    end=self._parse_end(item),
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Lemoore City Council"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+
+        title = (item.css("td:nth-child(1)::text").get()).strip()
+
+        if "SP" in title:
+            return "Lemoore City Council Special Meeting"
+        else:
+            return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return CITY_COUNCIL
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+
+        #css selector to get string that contains date and time
+        date_raw = (item.css("td:nth-child(1)::text").get()).strip()
+
+        #extract date from string
+        date = re.findall(r"\w+ \d{1,2}, \d{4}", date_raw)[0]
+
+        #meetings are at 5:30PM
+        time = "17:30:00"
```

Comment:
Right now, if you don't modify this with the regex below it stays a string, if you change this to a datetime you'll be good